packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -429,53 +399,63 @@ class Treemap extends StatelessWidget {
 
   Widget buildTitleText(BuildContext context) {
     if (isOutermostLevel) {
-      final pathFromRoot = rootNode.pathFromRoot();
-      // Build breadcrumbs navigator.
+      return buildBreadcrumbsNavigator();
+    } else {
       return Container(
         height: treeMapHeaderHeight,
-        child: ListView.separated(
-          shrinkWrap: true,
-          scrollDirection: Axis.horizontal,
-          separatorBuilder: (context, index) {
-            return const Text(' > ');
-          },
-          itemCount: pathFromRoot.length,
-          itemBuilder: (BuildContext context, int index) {
-            return buildSelectable(
-              child: Text(
-                index < pathFromRoot.length - 1
-                    ? pathFromRoot[index].name
-                    : pathFromRoot[index].displayText(),
-              ),
-              newRoot: pathFromRoot[index],
-            );
-          },
+        width: double.infinity,
+        decoration: BoxDecoration(
+          border: Border.all(color: Colors.black87),
         ),
-      );
-    } else {
-      return buildSelectable(
-        child: Container(
-          height: treeMapHeaderHeight,
-          width: double.infinity,
-          decoration: BoxDecoration(
-            border: Border.all(color: Colors.black87),
-          ),
-          child: buildNameAndSizeText(
-            textColor: Theme.of(context).textTheme.bodyText2.color,
-            oneLine: true,
-          ),
+        child: Text(
+          rootNode.displayText(),
+          style: TextStyle(color: Theme.of(context).textTheme.bodyText2.color),
+          textAlign: TextAlign.center,
+          overflow: TextOverflow.ellipsis,
         ),
       );
     }
   }
 
+  Container buildBreadcrumbsNavigator() {
+    final pathFromRoot = rootNode.pathFromRoot();
+    return Container(
+      height: treeMapHeaderHeight,
+      child: ListView.separated(
+        shrinkWrap: true,
+        scrollDirection: Axis.horizontal,
+        separatorBuilder: (context, index) {
+          return const Text(' > ');
+        },
+        itemCount: pathFromRoot.length,
+        itemBuilder: (BuildContext context, int index) {
+          return buildSelectable(
+            child: Text(
+              index < pathFromRoot.length - 1
+                  ? pathFromRoot[index].name
+                  : pathFromRoot[index].displayText(),
+            ),
+            newRoot: pathFromRoot[index],
+          );
+        },
+      ),
+    );
+  }
+
   /// Builds a selectable container with [child] as its child.
   ///
   /// Selecting this widget will trigger a re-root of the tree
   /// to the associated [TreemapNode].
   ///
   /// The default value for newRoot is [rootNode].
-  Tooltip buildSelectable({@required Widget child, TreemapNode newRoot}) {
+  Widget buildSelectable({
+    @required Widget child,
+    TreemapNode newRoot,
+    bool selectable = true,
```

Comment:
if it is not selectable, you are just returning child. So at the call site where you are passing selectable: false, don't call `buildSelectable` and just use the child directly when that condition is met. then you can remove the selectable param from this method, which is surprising since a "selectable" should always be selectable, intuitively.